src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -1,266 +1,266 @@
-OBA:VT0008897	blood immunoglobulin G2c amount	PR:000050261	IgG2c heavy chain protein	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0010474		CHEBI:50404	lipoprotein cholesterol	PATO:0000025	composition	UBERON:0000178	blood			
-OBA:VT0000203	blood aspartate transaminase amount	PR:000008153	aspartate aminotransferase, cytoplasmic	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0005119	blood thyroid-stimulating hormone amount	PR:000028269	thyroid stimulating hormone	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0008550	blood interferon-beta amount	PR:000024939	interferon beta	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0008496	blood immunoglobulin G2a amount	PR:000050259	IgG2a heavy chain protein	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0003457	blood ketone body amount	CHEBI:73693	ketone body	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0010476	blood glutamate dehydrogenase amount	GO:1990148	glutamate dehydrogenase complex	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:VT0010540	urine creatinine amount	CHEBI:16737	creatinine	PATO:0000070	amount	UBERON:0001088	urine			
-OBA:VT0010714	milk zinc content	CHEBI:27363	zinc atom	PATO:0000070	amount	UBERON:0001913	milk			
-OBA:2050054	blood chromium amount	CHEBI:28073	chromium atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050055	serum iron amount	CHEBI:18248	iron atom	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050056	serum homoarginine amount	CHEBI:24606	homoarginine	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050057	serum galactose-deficient IgA1 amount	PR:000050159	IgA1 immunoglobulin complex, circulating (human)	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050059	serum selenium amount	CHEBI:27568	selenium atom	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050060	serum dimethylarginine amount	CHEBI:86468	dimethylarginine	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050061	blood manganese amount	CHEBI:18291	manganese atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050062	serum alanine aminotransferase amount	PR:000050219	alanine aminotransferase	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050063	blood aluminium amount	CHEBI:28984	aluminium atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050064	blood nickel amount	CHEBI:28112	nickel atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050065	serum urea amount	CHEBI:16199	urea	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050066	blood molybdenum amount	CHEBI:28685	molybdenum atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050067	blood cadmium amount	CHEBI:22977	cadmium atom	PATO:0000070	amount	UBERON:0000178	blood			
-OBA:2050068	serum albumin amount	PR:000003918	albumin	PATO:0000070	amount	UBERON:0001977	serum			
-OBA:2050070	serum IgM amount	GO:0071754	IgM immunoglobulin complex, circulating	PATO:0000070	amount	UBERON:0001977	serum			
+OBA:0000016	angiotensin levels in blood	PR:000036009	angiotensin II	PATO:0000070	amount	UBERON:0000178	blood
```

Comment:
The tab separators for the last columns are missing here and at many similar places.

Reply:
There are still tab characters missing here and elsewhere in the tsv file.